src/kudos/services/trello.service.ts

```diff
@@ -0,0 +1,145 @@
+import {Injectable, HttpService} from '@nestjs/common';
+import * as TrelloNodeAPI from 'trello-node-api';
+import {InjectConfig} from 'nestjs-config';
+import {UserService} from "./user.service";
+import {map} from "rxjs/operators";
```

Comment:
I think linter was supposed to ask for keeping an order in these, are we using here one?